08-sceptre-user-app.md

```diff
@@ -124,8 +124,8 @@ The SCEPTRE app is configured by specifying the desired configurations in a scen
 ### SCADA Server
 
 - `type: scada-server`
-- `project`: Project file for mySCADA software.
-- `automation`: Automation file for mySCADA software start up.
+- `project`: Project file for mySCADA software. If no project file is given, a project file will be created using the OPC config to create a basic HMI with a list of all tag names and associated tag values. 
+- `automation`: Automation file for mySCADA software start up. If no automation file is given, an alternative method of automation described in the `scada.mako` will be used, but this auotmation may have issues depending on VM screen resolution depending on your build. 
```

Comment:
Can you provide a link to scada.mako, or at least a reference to what codebase it's in?